gramjs/Utils.js

```diff
@@ -281,24 +404,306 @@ function getInputMessage(message) {
             return new types.InputMessageID(message.id)
         }
         // eslint-disable-next-line no-empty
-    } catch (e) {
-    }
+    } catch (e) {}
 
     _raiseCastFail(message, 'InputMessage')
 }
 
+function getInputDocument(document) {
```

Comment:
document is a reserved word in javascript so maybe it should be changed to `doc` instead